R/reportEmployment.R

```diff
@@ -11,15 +10,16 @@
 #' @author Aman Malik
 #' @examples
 #' \dontrun{
-#' reportEmployment(gdx, improvements = "All", multiplier = "own", subtype = "expert", 
-#' share_manf = "local", decline = "capcosts",type="per_year")
+
+#' reportEmployment(gdx, improvements = "All", multiplier = "own", subtype = "expert", share_manf = "local", decline = "capcosts")
```

Comment:
```suggestion
#' reportEmployment(gdx, improvements = "All", multiplier = "own", subtype = "expert",
#'                  share_manf = "local", decline = "capcosts")
```

Reply:
Examples must not have lines with more than 100 characters, or they will get a NOTE which prevents buildLibrary from successfully finishing.